keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/DefaultBaseImageCache.kt

```diff
@@ -0,0 +1,16 @@
+package com.netflix.spinnaker.keel.bakery
+
+import com.netflix.spinnaker.keel.bakery.api.BaseLabel
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+class DefaultBaseImageCache(
+  private val baseImages: Map<String, Map<String, String>>
+) : BaseImageCache {
+  override fun getBaseImage(os: String, label: BaseLabel) =
+    baseImages[os]?.get(label.name.toLowerCase()) ?: throw UnknownBaseImage(os, label)
+}
+
+@ConfigurationProperties("bakery.base-images")
+class BaseImageCacheProperties {
+  var baseImages: Map<String, Map<String, String>> = emptyMap()
+}
```

Comment:
Default implementation of `BaseImageCache` requires you to set the images up in configuration. This is what Rosco does as well so it should be adequate for now.